projects/robots/festo/robotino3/protos/Robotino3.proto

```diff
@@ -4,6 +4,34 @@
 # documentation url: https://www.cyberbotics.com/doc/guide/robotino3
 # This is a PROTO file for Webots for the robot Robotino 3 from Festo.
 # It is the Robotino 3 robot that can be extended with different features to fullfil your requirement.
+#
+# changed by Thomas Feldmeier 2020, servicerobotik-ulm.de:
+#
+# Robotino3 had a lot of bugs in movement, was not able to move fast.
+# 
+# proto/Robotino3.proto:
+#   changed robots coordinate system to x=front, y=left, z=up
+#   corrected position of wheels
+#   robot > Physics > centerOfMass 0 0 -0.1
+#
+# proto/Robotino3Wheel.proto:
+#   device > RotationalMotor > maxVelocity 137.46 (allows max. speed 10 m/s forwards)
+#   boundingObject > Cylinder > height 0.01  (from 0.06, better robot rotation)
+#
+# controller/robotino3.c, base.c, base.h:
+#   increased max speed from 0.1 m/s to 8 m/s
+#   increased demo movement speed from 0.05 m/s to 2 m/s
+#   corrected the movement calculation
+#   added acceleration
+#
+# worlds/robotino3.wbt:
+#   WorldInfo > ContactProperties >
+#     material1 "WheelMat"
+#     coulombFriction 0 2 0
+#     bounce 0
+#
+# DON'T USE IN WORLDS WITH WorldInfo.coordinateSystem "ENU"
+# (maybe the physics engine has an error in asymmetric friction)
```

Comment:
This is too much information to put in the PROTO file, we can get rid of the change log here, as history can be found on your PR. I would remove this text, and just keep your name as a contributor. The warning about "ENU" should go the documentation page.